src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpointIT.java

```diff
@@ -100,6 +153,8 @@ public void setUp() throws IOException {
    */
   @Test
   public void shouldCreateCollectionExercise() throws Exception {
+    stubCreateActionPlan();
+
     log.info("************ Starting shouldCreateCollectionExercise *************");
```

Comment:
I'd probably have refactored these lines out (or changed them to log.debug) but it's not super important.